custom_components/mqtt_vacuum_camera/options_flow.py

```
@@ -49,6 +49,35 @@
     COLOR_ZONE_CLEAN,
     CONF_ASPECT_RATIO,
     CONF_AUTO_ZOOM,
+    TRIM_ACTION_DELETE,
```

Ruff (0.8.2)

`.const.TRIM_ACTION_DELETE` imported but unused

Remove unused import

(F401)